t/06-users.t

```diff
@@ -23,7 +23,7 @@ subtest 'new users are not ops and admins' => sub {
 };
 
 subtest 'system user presence' => sub {
-    my $system_user = $t->app->schema->resultset("Users")->search({username => 'system'})->first;
+    my $system_user = $t->app->schema->resultset("Users")->system;
```

Comment:
that `system` method returns a `Users` object with only the `id` attribute set (because of the select). That's for performance.

The `is_admin` and `is_operator` below check won't work as expected because of this. They might be passing coincidentally.

Reply:
I see. I changed the "system" method to take an optional attribute hash and only supply that where applicable.